prow/jobs/kyma/components/remote-environment-broker/remote-environment-broker.yaml

```diff
@@ -0,0 +1,49 @@
+job_template: &job_template
+  optional: true
+  skip_report: true
+  decorate: true
+  path_alias: github.com/kyma-project/kyma
+  max_concurrency: 10
+  extra_refs:
+  - org: kyma-project
+    repo: test-infra
+    base_ref: master
+    path_alias: github.com/kyma-project/test-infra
+  spec:
+    containers:
+    - image: eu.gcr.io/kyma-project/prow/test-infra/buildpack-golang:v20181119-afd3fbd
+      securityContext:
+        privileged: true
+      command:
+      - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/pipeline.sh"
+      env:
+      - name: SOURCES_DIR
+        value: "/home/prow/go/src/github.com/kyma-project/kyma/components/remote-environment-broker"
+
+job_labels_template: &job_labels_template
+  preset-dind-enabled: "true"
+  preset-sa-gcr-push: "true"
+  preset-docker-push-repository: "true"
+
+presubmits: # runs on PRs
+  kyma-project/kyma:
+  - name: &name prow/kyma/components/remote-environment-broker
+    run_if_changed: "^components/remote-environment-broker/"
+    context: *name
+    branches:
+      - master
+    <<: *job_template
+    labels:
+      <<: *job_labels_template
+      preset-build-pr: "true"
+
+postsubmits:
+  kyma-project/kyma:
+    - name: &name prow/kyma/components/remote-environment-broker
+      context: *name
```

Comment:
we decided to not specify a context in configuration, because when it is not configured, job name is used.
Please remove it (remember about removing assertion from test)

Reply:
Ok, I deleted it from this file and from binding-usage-controller.yaml. Tests are adjusted.